.studiorc

```diff
@@ -18,15 +18,11 @@ function install_from_source() {
 }
 
 function lint() {
-  pushd /src > /dev/null
-    shellcheck -s bash bin/* lib/*
-  popd > /dev/null
+  shellcheck -e SC1117 -s bash /src/bin/* /src/dot-studio/*
```

Comment:
If it's not totally impractical, I'd suggest omitting the `-s bash` in favor of adding `#!/bin/bash` to the files themselves. This has the benefit of letting editors apply shellcheck correctly when someone is working on the file.

Also, any reason not to fix up the [SC1117](https://github.com/koalaman/shellcheck/wiki/SC1117) instances? It's definitely a stylistic issue and therefore ignorable, but upon reading about it, I realize I had a subtle misunderstanding of what the shell actually does here that could bite me in other instances. Not a big deal, but just thought it was worth considering.